Scripts/Runtime/Entities/Tasks/Data/UpdateJobData.cs

```diff
@@ -0,0 +1,41 @@
+using Unity.Core;
+using Unity.Entities;
+
+namespace Anvil.Unity.DOTS.Entities
+{
+    public class UpdateJobData<TData>
+        where TData : unmanaged, IProxyData
+    {
+        private readonly ProxyDataStream<TData> m_UpdateProxyDataStream;
+
+        public World World
+        {
+            get;
+        }
+
+        public ref readonly TimeData Time
+        {
+            get => ref World.Time;
+        }
+
+        public byte Context
+        {
+            get;
+        }
+
+        public UpdateJobData(World world, 
+                             byte context, 
+                             ProxyDataStream<TData> updateProxyDataStream)
+        {
+            World = world;
+            Context = context;
+            m_UpdateProxyDataStream = updateProxyDataStream;
+        }
+
+        public PDSUpdater<TData> GetPSDUpdater()
+        {
+            PDSUpdater<TData> updater = m_UpdateProxyDataStream.CreateVDUpdater(Context);
```

Comment:
- Missed updating the CreateVDUpdater method to PD.
- PDS is too abbreviated for my preference. `PDStreamUpdater` or `DataStreamUpdater` would be easier to understand.

```suggestion
            PDStreamUpdater<TData> updater = m_UpdateProxyDataStream.CreatePDStreamUpdater(Context);
```

Reply:
Cool, I like `DataStreamX` so updating all of these.